CHANGES.md

```diff
@@ -6,54 +6,44 @@
 
   * `XMonad.Prompt`
 
-    Now `mkComplFunFromList` and `mkComplFunFromList'` take an
-    additional `XPConfig` argument, so that they can take into
-    account the given `searchPredicate`.
+    - Now `mkComplFunFromList` and `mkComplFunFromList'` take an
+      additional `XPConfig` argument, so that they can take into
+      account the given `searchPredicate`.
 
-    A `complCaseSensitivity` field has been added to `XPConfig`, indicating
-    whether case-sensitivity is desired when performing completion.
-
-  * `XMonad.Hooks.EwmhDesktops`
-
-    It is no longer recommended to use `fullscreenEventHook` directly.
-    Instead, use `ewmhFullscreen` which additionally advertises fullscreen
-    support in `_NET_SUPPORTED` and fixes fullscreening of applications that
-    explicitly check it, e.g. mupdf-gl, sxiv, …
-
-    `XMonad.Layout.Fullscreen.fullscreenSupport` now advertises it as well,
-    and no configuration changes are required in this case.
+    - A `complCaseSensitivity` field has been added to `XPConfig`, indicating
+      whether case-sensitivity is desired when performing completion.
 
+    - `historyCompletion` and `historyCompletionP` now both have an `X`
+      constraint (was: `IO`), due to changes in how the xmonad core handles XDG
+      directories.
 
   * `XMonad.Hooks.EwmhDesktops`
 
-    `ewmh` function will use `logHook` for handling activated window. And now
-    by default window activation will do nothing.
+    - It is no longer recommended to use `fullscreenEventHook` directly.
+      Instead, use `ewmhFullscreen` which additionally advertises fullscreen
+      support in `_NET_SUPPORTED` and fixes fullscreening of applications that
+      explicitly check it, e.g. mupdf-gl, sxiv, …
 
-    You can use regular `ManageHook` combinators for changing window
-    activation behavior and then add resulting `ManageHook` using
-    `activateLogHook` to your `logHook`. Also, module `X.H.Focus` provides
-    additional combinators.
+    - `XMonad.Layout.Fullscreen.fullscreenSupport` now advertises it as well,
```

Comment:
This is a continuation of the preceding paragraph, and so is the "You can use regular…"
See https://github.com/xmonad/xmonad-contrib/pull/399/commits/c086a49d470daa0f72cbdeaf6dc0b993a9b5fb76
(feel free to cherry-pick or just drop the two dashes)

Reply:
Ah right, should've read more carefully; so that's why these two heading were there.